From: Simon McVittie <smcv@debian.org>
Date: Mon, 16 Nov 2020 23:13:03 +0000
Subject: tests: Mark pixbuf-randomly-modified as flaky

This is basically a crude fuzzer. It isn't really suitable for
build-time acceptance testing, since it frequently produces images for
which gdk-pixbuf will try to allocate more memory than is available,
but if it does, it isn't straightforward to capture the failing image
from an autobuilder that only records build logs.

Bug: https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/issues/146
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=942124
Signed-off-by: Simon McVittie <smcv@debian.org>
---
 tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index c0ecf30..cf58db1 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -79,7 +79,7 @@ installed_tests = {
   },
   'pixbuf-fail': { 'suites': ['conform', 'slow'], },
   'pixbuf-icon-serialize': { 'suites': ['conform'], },
-  'pixbuf-randomly-modified': { 'suites': ['slow'], },
+  'pixbuf-randomly-modified': { 'suites': ['slow', 'flaky'], },
   'pixbuf-threads': { 'suites': ['io'], },
   'pixbuf-gif': {
     'suites': ['io'],
